x-pack/plugins/ml/public/application/services/ml_api_service/data_frame_analytics.ts

```diff
@@ -166,4 +166,12 @@ export const dataFrameAnalytics = {
       method: 'GET',
     });
   },
+  validateDataFrameAnalytics(analyticsConfig: DeepPartial<DataFrameAnalyticsConfig>) {
+    const body = JSON.stringify(analyticsConfig);
+    return http<any>({
```

Comment:
can we replace `any` with the return type here?
should be the return type of `validateAnalyticsJob`

Reply:
Updated in 943882adedeeaf1254d355f87d7b8549051e6c37